client/chess.ts

```diff
@@ -5,6 +5,8 @@ import * as util from 'chessgroundx/util';
 import { read } from 'chessgroundx/fen';
 
 import { _ } from './i18n';
+import {Color, Key, Pieces, Role} from "chessgroundx/types";
+import {InsertHook} from "snabbdom/src/hooks";
 
 const pieceSan = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z'] as const;

```

Comment:
I actually think the types defined here should be more sophisticated. We should talk on Discord about this matter.